kv/error.go

```diff
@@ -14,44 +14,33 @@
 package kv
 
 import (
-	"strings"
-
 	"github.com/pingcap/parser/mysql"
 	"github.com/pingcap/parser/terror"
 )
 
 // KV error codes.
 const (
-	codeClosed                     terror.ErrCode = 1
-	codeNotExist                                  = 2
-	codeConditionNotMatch                         = 3
-	codeLockConflict                              = 4
-	codeLazyConditionPairsNotMatch                = 5
-	codeRetryable                                 = 6
-	codeCantSetNilValue                           = 7
-	codeInvalidTxn                                = 8
-	codeNotCommitted                              = 9
-	codeNotImplemented                            = 10
-	codeTxnTooLarge                               = 11
-	codeEntryTooLarge                             = 12
-
-	codeKeyExists = 1062
+	codeClosed          terror.ErrCode = 1
+	codeNotExist                       = 2
+	codeRetryable                      = 6
+	codeCantSetNilValue                = 7
+	codeInvalidTxn                     = 8
+	codeNotImplemented                 = 10
+	codeTxnTooLarge                    = 11
+	codeEntryTooLarge                  = 12
+	codeKeyExists                      = 1062
 )
 
+// TxnRetryableMark is used to uniform the commit error messages which could retry the transaction.
+const TxnRetryableMark = " [try again later]"
+
 var (
 	// ErrClosed is used when close an already closed txn.
 	ErrClosed = terror.ClassKV.New(codeClosed, "Error: Transaction already closed")
 	// ErrNotExist is used when try to get an entry with an unexist key from KV store.
 	ErrNotExist = terror.ClassKV.New(codeNotExist, "Error: key not exist")
-	// ErrConditionNotMatch is used when condition is not met.
-	ErrConditionNotMatch = terror.ClassKV.New(codeConditionNotMatch, "Error: Condition not match")
-	// ErrLockConflict is used when try to lock an already locked key.
-	ErrLockConflict = terror.ClassKV.New(codeLockConflict, "Error: Lock conflict")
-	// ErrLazyConditionPairsNotMatch is used when value in store differs from expect pairs.
-	ErrLazyConditionPairsNotMatch = terror.ClassKV.New(codeLazyConditionPairsNotMatch, "Error: Lazy condition pairs not match")
-	// ErrRetryable is used when KV store occurs RPC error or some other
-	// errors which SQL layer can safely retry.
-	ErrRetryable = terror.ClassKV.New(codeRetryable, "Error: KV error safe to retry")
+	// ErrRetryable is used when KV store occurs RPC error or some other errors which SQL layer can safely retry.
```

Comment:
Better add comment
`currently, the err is returned when lock not found in Commit, subjective to change in the future.`